### PR TITLE
INT-7989 - omit duplicates

### PR DESCRIPTION
**Pull request by adam-in-ict**
# Description

Thank you for contributing to a JupiterOne integration!

Please include a summary of the change and which issue is fixed. Please also
include relevant motivation and context. List any dependencies that are required
for this change.

## Summary

Omitting duplicate data sources (confirmed all properties match on duplicates).

## Type of change

Please leave any irrelevant options unchecked.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### General Development Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### Integration Development Checklist:

Please leave any irrelevant options unchecked.

- [ ] I have checked for additional permissions required to call any new API
      endpoints, and have documented any additional permissions in
      `jupiterone.md`, where necessary.
- [ ] My changes properly paginate the target service provider's API
- [ ] My changes properly handle rate limiting of the target service provider's
      API
- [ ] My new integration step is instrumented to execute in the correct order
      using `dependsOn`
- [ ] I have referred to the
      [JupiterOne data model](https://github.com/JupiterOne/data-model/tree/main/src/schemas)
      to ensure that any new entities/relationships, and relevant properties,
      match the recommended model for this class of data
- [ ] I have updated the `CHANGELOG.md` file to describe my changes
- [ ] When changes include modifications to existing graph data ingestion, I've
      reviewed all existing managed questions referencing the entities,
      relationships, and their property names, to ensure those questions still
      function with my changes.

### CODE REVIEW

File: test/recording.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function setupProjectRecording(
 
 function getRedactedTokenResponse(original) {
   const payload = JSON.parse(original);
-  payload.auth_token = '[REDACTED]';
+  payload.systemToken = '[REDACTED]';
```
Comment (VDubber) [inline test/recording.ts:25]: 😨

Comment (adam-in-ict) [reply test/recording.ts:25]: We were previously correctly redacting it, but missed it when switching to user tokens.  These were all short lived tokens strictly for the public sandbox account.

File: CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,3 @@
-* @jupiterone/integrations-staff @jupiterone/integrations-contractors
+* @jupiterone/integrations
```
Comment (VDubber) [inline CODEOWNERS:1]: Do we already have a desk ticket created?

Comment (adam-in-ict) [reply CODEOWNERS:1]: I checked with the current Security on-call and was told I didn't need one for CODEOWNER changes.

**Pull request merged**